### PR TITLE
April TFE release small updates 

### PR DESCRIPTION
**Pull request by rkoron007**
Small fixes from the April TFE release release: https://github.com/hashicorp/ptfe-releases/pull/1160

### CODE REVIEW

File: website/docs/cloud-docs/api-docs/index.mdx [hunk 1]
```diff
@@ -83,7 +83,7 @@ The following entitlements are available:
 <!-- END: TFC:only name:tfc-audit-log -->
 - `configuration-designer` — Allows an organization to use the [Configuration Designer][].
 - `cost-estimation` — Allows an organization to access [cost estimation][].
-- `global-run-tasks` — Allows an organization apply [run tasks](/terraform/cloud-docs/workspaces/settings/run-tasks) at the organization level. Affects the [run tasks][] endpoints. This feature is currently in beta.
+- `global-run-tasks` — Allows an organization to apply [run tasks](/terraform/cloud-docs/workspaces/settings/run-tasks) to every workspace in an organization. Affects the [run tasks][] endpoints. This feature is currently in beta.
```
Comment (ritsok) [inline website/docs/cloud-docs/api-docs/index.mdx:86]:
```suggestion
- `global-run-tasks` — Allows an organization to apply [run tasks](/terraform/cloud-docs/workspaces/settings/run-tasks) to every workspace. Affects the [run tasks][] endpoints. This feature is currently in beta.
```

File: website/docs/cloud-docs/api-docs/oauth-clients.mdx [hunk 2]
```diff
@@ -456,13 +456,13 @@ curl \
   https://app.terraform.io/api/v2/oauth-clients/oc-XKFwG6ggfA9n7t1K
 ```
 
-## Attach an OAuth Client to projects
+## Attach to a project
```
Comment (ritsok) [inline website/docs/cloud-docs/api-docs/oauth-clients.mdx:459]: i do feel like associate is the more appropriate verb here, but can revisit later. maybe something to keep in mind as we build out adam's controlled vocab list

File: website/docs/cloud-docs/workspaces/settings/run-tasks.mdx [hunk 3]
```diff
@@ -50,7 +50,7 @@ To create a new run task:
 
 ### Global Run Tasks
 
-When you create a new run task you may choose to apply to all workspaces in the organization. This feature requires the **global-run-task** [entitlement][].
+When you create a new run task, you can choose to apply it globally to every workspace in an organization. This feature requires the **global-run-task** [entitlement][].
```
Comment (ritsok) [inline website/docs/cloud-docs/workspaces/settings/run-tasks.mdx:53]: last sentences read a little funny - i assume the entitlement is part of a paid plan maybe? feature enabling an entitlement feels confusing, maybe im misunderstanding?

**Review by ritsok — APPROVED** [review]

File: website/docs/cloud-docs/workspaces/settings/run-tasks.mdx (hunk 3 above)
Comment (rkoron007) [reply website/docs/cloud-docs/workspaces/settings/run-tasks.mdx:53]: Thank you for reminding me I wanted to look up how entitlements work! 

>Each organization has a set of entitlements that corresponds to its pricing tier. These entitlements determine which HCP Terraform features the organization can use.
>
> If an organization doesn't have the necessary entitlement to use a given feature, HCP Terraform returns a 404 error for API requests to any endpoints devoted to that feature.

So it sounds like you need entitlements to unlock specific features for an organization. Does this feel like a better way of putting this?
```suggestion
When you create a new run task, you can choose to apply it globally to every workspace in an organization. Your organization must have the `global-run-task` [entitlement][] to use global run tasks.
```

**Pull request merged**